httime/httime.go

```diff
@@ -10,10 +10,24 @@ import (
 )
 
 const (
-	StrftimeChar     = "%"
-	UnixTimestampFmt = "%s(%L)?"
+	StrftimeChar        = "%"
+	UnixTimestampFmt    = "%s(%L)?"
+	UnixTimestampFmtAlt = "%s(.%L)?"
+	UnixTimestampFmtTxt = "unixepoch"
 )
 
+// TODO UnixTimestampFmt is not a regex - it shouldn't look like one. Also even
+// if it were a regex it would be wrong because it's expecting a dot between the
+// seconds and fractional seconds, eg `%s(.%L)?`. But just changing it would
+// break anybody that is using the magical  string `%s(%L)?` as their timestamp
+// in order to enable this magical behavior. Is anybody doing that? Who can
+// know? Introducing UnixTimestampFmtAlt gets it closer to what it's actually
```

Comment:
> Who can know?

This comment is headed in a dangerously existential (well, epistemological really) direction :D